nodepool/nl02.sjc1.vexxhost.zuul.ansible.com.yaml

```diff
@@ -214,6 +219,16 @@ providers:
               - public
               - net1
               - net2
+          - name: vqfx-18.1R3
+            flavor-name: v2-highcpu-1
+            cloud-image: vqfx-18.1R3-S2.5-20200116
+            key-name: infra-root-keys
+            boot-from-volume: true
+            volume-size: 20
```

Comment:
This can be 80GB, per other appliances. We can review all others together if we want each to be different

Reply:
I think this one will just work with 20GB, +1 on the review. It'll help us save resource. Will change this one to 80Gb for now just to be consistent with another platforms until the change.